Source/AliveLibAO/Abe.cpp

```diff
@@ -6882,11 +6883,284 @@ void Abe::State_61_Respawn_42CD20()
+            gElum_507680->field_10_anim.field_4_flags.Set(AnimFlags::eBit5_FlipX, pSaveData->field_272_elum_flag & 1);
+            gElum_507680->field_E6_last_anim_frame = 0;
+            gElum_507680->field_120 = 1;
+            gElum_507680->field_122 = pSaveData->field_276_elum_122;
+            gElum_507680->field_128_brain_idx = pSaveData->field_278_brain_idx;
+            gElum_507680->field_12A_brain_state = pSaveData->field_27A_elum_brain_state;
+            gElum_507680->field_12C_honey_xpos = static_cast<short>(pSaveData->field_27C_honey_xpos);
+            gElum_507680->field_146 = pSaveData->field_280_honey_ypos;
+            gElum_507680->field_130 = pSaveData->field_284_elum_field_130;
+
+            gElum_507680->field_170_flags.Set(Elum::Flags_170::eFoundHoney_Bit4, pSaveData->field_28B & 1);
+            gElum_507680->field_170_flags.Set(Elum::Flags_170::eBit3, pSaveData->field_28A_some_flag & 1);
+            gElum_507680->field_170_flags.Set(Elum::Flags_170::eBit2, pSaveData->field_289 & 1);
+            if (gElum_507680->field_B0_path_number == sActiveHero_507678->field_B0_path_number)
+            {
+                if (pSaveData->field_270 != -1)
+                {
+                    PathLine* pLine = nullptr;
+                    FP hitX = {};
+                    FP hitY = {};
+                    if (sCollisions_DArray_504C6C->RayCast_40C410(
+                        gElum_507680->field_A8_xpos,
+                        gElum_507680->field_AC_ypos - FP_FromInteger(60),
+                        gElum_507680->field_A8_xpos,
+                        gElum_507680->field_AC_ypos + FP_FromInteger(60),
+                        &pLine,
+                        &hitX,
+                        &hitY,
+                        1 << pSaveData->field_270))
+                    {
+                        gElum_507680->field_F4_pLine = pLine;
+                        gElum_507680->field_FC_current_motion = eElumStates::State_1_Idle_412990;
+                        gElum_507680->field_E4_previous_motion = eElumStates::State_1_Idle_412990;
+                    }
+                    else
+                    {
+                        gElum_507680->field_FC_current_motion = eElumStates::State_21_Land_414A20;
+                        gElum_507680->field_E4_previous_motion = eElumStates::State_21_Land_414A20;
+                    }
+                }
+            }
+            else
+            {
+                gElum_507680->field_F4_pLine = (PathLine *) -2;
+            }
+            if (gElum_507680->field_170_flags.Get(Elum::Flags_170::eFoundHoney_Bit4))
+            {
+                if (!ResourceManager::GetLoadedResource_4554F0(ResourceManager::Resource_Animation, 203, 0, 0))
+                {
+                    ResourceManager::LoadResourceFile_455270("ELMHONEY.BAN", 0);
+                }
+                gElum_507680 = gElum_507680;
```

Comment:
:)

Reply:
...sometimes it's important to call a spade a spade :tape: